exec/vector/src/main/codegen/templates/UnionVectorListWriter.java

```diff
@@ -0,0 +1,212 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import org.apache.drill.exec.vector.complex.DictVector;
+import org.apache.drill.exec.vector.complex.ListVector;
+import org.apache.drill.exec.vector.complex.MapVector;
+import org.apache.drill.exec.vector.complex.impl.UnionVectorWriter;
+import org.apache.drill.exec.vector.complex.writer.FieldWriter;
+
+<@pp.dropOutputFile />
+<@pp.changeOutputFile name="/org/apache/drill/exec/vector/complex/impl/UnionVectorListWriter.java" />
+<#include "/@includes/license.ftl" />
+package org.apache.drill.exec.vector.complex.impl;
+
+<#include "/@includes/vv_imports.ftl" />
+
+import java.util.EnumMap;
+import java.util.Map;
+import java.util.function.Function;
+
+/**
+ * Union vector writer for writing list of union-type values
+ */
+public class UnionVectorListWriter extends UnionVectorWriter {
+
+  private final ListVector listVector;
+  private final UInt4Vector offsets;
+  private int listPosition;
+
+  public UnionVectorListWriter(ListVector listVector, FieldWriter parent) {
+    super(listVector.promoteToUnion(), parent);
+    this.listVector = listVector;
+    this.offsets = listVector.getOffsetVector();
+  }
+
+  //===============================FACTORIES FOR COMPLEX TYPE WRITERS=============================================================
```

Comment:
I would suggest to remove `==============`, just leave simple comment:
// FACTORIES FOR COMPLEX TYPE WRITERS

Please consider the same for other cases below.

Reply:
done.